cmd/juju/application/addunit_test.go

```diff
@@ -66,6 +66,17 @@ func (f *fakeApplicationAddUnitAPI) AddUnits(args apiapplication.AddUnitsParams)
 	return nil, nil
 }
 
+func (f *fakeApplicationAddUnitAPI) ScaleApplication(args apiapplication.ScaleApplicationParams) (params.ScaleApplicationResult, error) {
```

Comment:
As with destroy, we need test(s) similar to TestDestroyUnknownModelCallsRefresh etc